addons/base_import/models/base_import.py

```diff
@@ -937,6 +946,16 @@ def parse_preview(self, options, count=10):
                         matches[index] = match.split('/')
             elif options.get('has_headers'):
                 matches = self._get_mapping_suggestions(headers, header_types, fields_tree)
+                if self._context.get("lang"):
+                    guessed_lang, guessed_lang_term_ratio = self.env['ir.translation']._guess_language_of_terms(
+                        terms=[re.sub(COLUMN_FIELD_HINT_REGEX, '', header)
+                               for header_full in headers for header in header_full.split('/')],
+                        types={'model'})
+                    if guessed_lang_term_ratio > 0.5 and self._context.get("lang") != guessed_lang:
```

Comment:
Why 0.5 ? Can't it be a global so that we may tweak it ?

Reply:
Yes that is a good idea, I have added a constant at the top of the file (with the other ones) with a comment.